packages/slice-machine/tests/src/utils/fieldNameCreator.test.ts

```diff
@@ -0,0 +1,51 @@
+import "@testing-library/jest-dom";
+
+import { expect } from "@jest/globals";
+
+import { createFriendlyFieldNameWithId } from "@src/utils/fieldNameCreator";
+
```

Comment:
OPT: a test to verify that it works with more than 2 words (new object in the array)